pilot/pkg/config/clusterregistry/multicluster.go

```diff
@@ -40,6 +41,7 @@ type Multicluster struct {
 	serviceController *aggregate.Controller
 	ClearCache        func()
 	rkc               map[string]*kubeController
+	sync.Mutex
```

Comment:
Do you want to protect `rkc`? Then you should move sync.Mutex up before rkc

Reply:
I will move this as requested just to avoid argument, but why does the order matter?  Is this a style thing.   https://tour.golang.org/concurrency/9 indicates it should be after the variable you will need to protect.

Reply:
We always make critical section grouped together and the lock before, I did not see any official doc about this, but we should obey the same coding style in the same repo at least.